tests/scripts/format.py

```diff
@@ -227,6 +227,13 @@ def main(argv):
     proc = subprocess.Popen(["dotnet", "build-server", "shutdown"], env=my_env)
     output,error = proc.communicate()
 
+    # shutdown all spurious dotnet processes using os shell
+    if platform == 'Linux' or platform == 'OSX':
+        subprocess.call(['killall', '-SIGTERM', '-qw', dotnet])
+    elif platform == 'Windows_NT':
+        utilpath = os.path.join(coreclr, 'tests\\scripts\\utilities.cmd')
```

Comment:
"utilities.cmd" seems too generic. Why not "kill-all.cmd" or "kill-all-named-processes.cmd"?